src/js/actions/getEvents.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+import { FETCH_EVENTS_SUCCESS } from "./types";
+
+export const getEventsAction = () => dispatch => {
+  return axios
```

Comment:
@magicmarie You should create an axios instance, then call the different http verb methods from it